### PR TITLE
Removed unnecessary vendors, changed path from FrontEnd to packages

### PR DESCRIPTION
**Pull request by ptsavdar**
Fixing #158

### CODE REVIEW

File: plugin/portfolio/Resources/views/Portfolio/index.html.twig [hunk 1]
```diff
@@ -96,7 +96,7 @@
       '@IcapPortfolioBundle/Resources/views/less/comments.less'
       '@IcapPortfolioBundle/Resources/views/less/statistics.less'
       '@IcapPortfolioBundle/Resources/views/less/daterangepicker.less'
-      '@FrontEndBundle/Resources/public/toastr/toastr.less'
+      'packages/toastr/toastr.less'
```
Comment (LaurentGruber) [inline plugin/portfolio/Resources/views/Portfolio/index.html.twig]: is it not better to compile less file in the installation process (or in the package creation process) and reference the compiled file in the template ?
@claroline/maintainers

Comment (ptsavdar) [inline plugin/portfolio/Resources/views/Portfolio/index.html.twig]: The files are compiled during installation/update process then minified and stored in the web folder (as css files). The app then loads these files every time it is called.

Comment (LaurentGruber) [inline plugin/portfolio/Resources/views/Portfolio/index.html.twig]: Ok thanks. LGTM

**Comment by stefk** [comment]
LGTM

**Comment by LaurentGruber** [comment]
LGTM

**Pull request merged**